### ISSUE TITLE
Adding support for website

### ISSUE THREAD

**Issue by J0shua-code** [ytdl-org/youtube-dl#30093]
## Checklist

- [ x] I'm asking a question
- [ x] I've looked through the README and FAQ for similar questions
- [ x] I've searched the bugtracker for similar questions including closed ones

## Question

I am was looking to add support for downloading stories from snapchat (pages not public user stories) 
eg: https://story.snapchat.com/p/977d62f4-f1c1-45eb-bf10-63e6e750cd65/1111760915728384

But snapchat for some reason doesn't use .m3u8 or any other manifest files
instead it has a sequencial list of .mp4 files in json format in the webpage I can extract the links but I cant figure out how to make youtube-dl merge  them into a single video file

**Comment by dirkf** [comment]
Isn't that a playlist?

Then as long as the .mp4 files all have the same parameters:
`ffmpeg -i input1.mp4 -i input2.mp4 ... -a: copy -v:copy output.mp4`

**Comment by J0shua-code** [comment]
> Isn't that a playlist?
> 
> Then as long as the .mp4 files all have the same parameters: `ffmpeg -i input1.mp4 -i input2.mp4 ... -a: copy -v:copy output.mp4`

yeah ik, but i needed some method to do that in the extractor for adding support for snapchat

#https://github.com/ytdl-org/youtube-dl/pull/28153

but it looks like the playlist concat is not yet implemented
